test/plugin/scenarios/elasticsearch-6.x-scenario/bin/startup.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+home="$(cd "$(dirname $0)"; pwd)"
+
+java -jar ${agent_opts} -Dskywalking.plugin.elasticsearch.trace_dsl=true ${home}/../libs/elasticsearch-6.x-scenario.jar &
+
+# user for more time to collect data
+sleep 5
```

Comment:
Hi, are you sure need to ```sleep 5``` ？

Reply:
Yes, if there is no waiting, it will often lead to incomplete collection of spans.

Reply:
You should use ```healthCheck``` to make sure that ES is ready, not through sleep.

Reply:
Yes, please follow @arugal guidance. The health check service is designed for you to verify the es is ready. Sleep is not a safe option, as it is affected by CPU payload.

Reply:
We have fixed the problem that the agent unregistered to make the span incomplete, if ES is not ready, it should be a service error, please check.

Reply:
Sorry, there is a problem with the above explanation. The healthCheck is there. In fact, the reason is not that the ES startup time takes a few seconds, but because six requests are executed at one time in the test service, six segments are generated, and now In the JVM container, the waiting time for collection is 5s. Due to insufficient waiting time, the local information will result in incomplete collection of information, and the subsequent segments are not collected. I can normally collect it by adding a sleep to the wait above. And this local test success.  I just tested it, I removed `sleep 5`, and the test passed.

Reply:
Ok, now i will remove `sleep 5`

Reply:
@aderm Anything you need to do or wait, put those in your codes and add check logic. Don't use time to verify, it is unstable, we were facing this kind of issue multiple times.

Reply:
ok